gamla/higher_order_test.py

```diff
@@ -76,19 +76,28 @@ def wrapped(*args, **kwargs):
     return decorator
 
 
-def generate_getter_and_setter(
+@currying.curry
+def _generate_getter_and_setter_with_encoder_and_decoder(
```

Comment:
if we remove encode / decode support from  gamla, this test belongs next to the stores. Wdyt ?

Reply:
We already have tests there for encode and decode in redis, I think it's okay to have it also here as these are specific getter and setter functions